src/cdk/drag-drop/drag-events.ts

```diff
@@ -54,13 +54,13 @@ export interface CdkDragExit<T = any, I = T> {
 
 
 /** Event emitted when the user drops a draggable item inside a drop container. */
-export interface CdkDragDrop<T, O = T> {
+export interface CdkDragDrop<T, O = T, I = T> {
   /** Index of the item when it was picked up. */
   previousIndex: number;
   /** Current index of the item. */
   currentIndex: number;
   /** Item that is being dropped. */
-  item: CdkDrag;
+  item: CdkDrag<I>;
```

Comment:
I think this has to be `target: major` because `item` as changed from `CdkDrag<any>` to `CdkDrag<I>` which can break people's code.

Reply:
It's not worth going through the deprecation process for this. I'll default it to `any` instead,